src/util/strconv.h

```diff
@@ -177,7 +190,7 @@ static size_t unicode_tolower(char *encoded, size_t in_len) {
 }
 
 
-// strndup + unescape + fold
+// strndup + unescape + tolower
 static char *rm_normalize(const char *s, size_t len) {
```

Comment:
no need to update len here as in `tag_strtolower?`

Reply:
I think we don't need it. Since in `tag_strtolower` we update the contents of the input string, we also update its length, but for `rm_normalize` we duplicate the input string and return a new pointer.